### PR TITLE
Update integrations to include alternate Drupal option.

### PR DESCRIPTION
**Pull request by fenetikm**
This provides an alternate option for integrating Drupal and phpactor. Any questions, let me know.

### CODE REVIEW

File: doc/integrations.md [hunk 1]
```diff
@@ -63,3 +67,36 @@ code_transform.indentation: 2
...
+### Option 2: Add the modules into the Composer autoloader
+
+This option requires merging in any modules (Drupal, contrib, custom) into the Composer autoloader via a discovery mechanism offered by the [Drupal Autoloader](https://github.com/fenetikm/autoload-drupal) composer plugin.
+
+Full details are available in the README for this plugin, however, the short version is that you will need to put the following into your `composer.json` file:
+
+```json
+{
+    "require": {
+        "fenetikm/autoload-drupal": "dev-master"
```
Comment (dantleech) [inline doc/integrations.md]: Would be nice to suggest to tag this, as people will actually be adding it their projects and it `dev-master` cold break...

**Comment by dantleech** [comment]
Really like this :) I think it would be a good idea to tag your library (`0.1`? or even `1.0`?) as otherwise it introduces a bit of a risk to anybody including it in their project.

**Comment by fenetikm** [comment]
Updated to 0.1 👍

**Comment by dantleech** [comment]
Brilliant, thanks :)

**Pull request merged**